metsrw/fsentry.py

```diff
@@ -135,6 +134,20 @@ def __init__(self, path=None, label=None, use='original', type=u'Item',
         self.amdsecs = []
         self.dmdsecs = []
 
+    @classmethod
+    def dir(cls, label, children):
+        """Return ``FSEntry`` directory object."""
+        return FSEntry(
+            label=label, children=children, type=u"Directory", use=None)
+
+    @classmethod
+    def with_fptr(cls, label, type_, fptr):
```

Comment:
`from_fptr` since it's kind of conversion? Also fine with this naming if you prefer it, it just doesn't seem obvious to me.

These are good classmethod usage.